src/components/Personalization/createFetchDataHandler.js

```diff
@@ -9,21 +9,87 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
+import { defer, groupBy } from "../../utils";
+
+const DECISIONS_HANDLE = "personalization:decisions";
 
 export default ({
   prehidingStyle,
-  responseHandler,
+  showContainers,
   hideContainers,
-  mergeQuery
+  mergeQuery,
+  collect,
+  processPropositions,
+  createProposition,
+  pendingDisplayNotifications
 }) => {
-  return ({ decisionsDeferred, personalizationDetails, event, onResponse }) => {
+  return ({ cacheUpdate, personalizationDetails, event, onResponse }) => {
     if (personalizationDetails.isRenderDecisions()) {
       hideContainers(prehidingStyle);
```

Comment:
add else clause here with `showContainers`

Reply:
Fixed in #1058